packages/perennial-vaults/contracts/balanced/BalancedVault.sol

```diff
@@ -11,7 +11,7 @@ import "./BalancedVaultDefinition.sol";
  * @notice ERC4626 vault that manages a 50-50 position between long-short markets of the same payoff on Perennial.
  * @dev Vault deploys and rebalances collateral between the corresponding long and short markets, while attempting to
  *      maintain `targetLeverage` with its open positions at any given time. Deposits are only gated in so much as to cap
- *      the maximum amount of assets in the vault.
+ *      the maximum amount of assets in the vault. The long and short markets are expected o have the same oracle.
```

Comment:
typo: to

Reply:
Similar to below comment, technically oracle and payoff are separate, so we could specify both here.